app/views/components/show.html.erb

```diff
@@ -0,0 +1,39 @@
+<h1><%= @component.name %></h1>
+
+<table class="govuk-table">
+  <h3 class="govuk-heading-m">Certificates to upload</h3>
+  <thead class="govuk-table__head">
+    <tr class="govuk-table__row">
+      <th class="govuk-table__header" scope="col">Name</th>
+      <th class="govuk-table__header" scope="col">Type</th>
+    </tr>
+  </thead>
+  <tbody class="govuk-table__body">
+      <tr class="govuk-table__row">
+        <td class="govuk-table__cell" scope="row"><%= @component.name %></td>
+        <td class="govuk-table__cell"><%= @component.component_type %></td>
+      </tr>
+  </tbody>
+
+</table>
+<%= link_to 'Upload', new_component_certificate_path(@component), class: 'govuk-button' %>
+
+<% if @component_certificates.present? %>
+  <table class="govuk-table">
+    <h3 class="govuk-heading-m">Certificates currently in use</h3>
+    <thead class="govuk-table__head">
+      <tr class="govuk-table__row">
+        <th class="govuk-table__header" scope="col">Usage</th>
+        <th class="govuk-table__header" scope="col">Last updated</th>
+      </tr>
+    </thead>
+    <tbody class="govuk-table__body">
+      <% @component_certificates.each do |certificate| %>
+        <tr class="govuk-table__row">
+          <td class="govuk-table__cell" scope="row"><%= certificate.usage %></td>
+          <td class="govuk-table__cell"><%= certificate.updated_at.strftime("%e-%m-%Y %H:%M") %></td>
```

Comment:
we should move this date formatting method on the model.

Reply:
👍